src/lib/axl-explorations/src/components/games/LetterLauncherGameStoryPreview.tsx

```
@@ -263,8 +263,8 @@ export function LetterLauncherGameStoryPreview({
 
     const audioLetter = lettersToUse[Math.floor(Math.random() * lettersToUse.length)];
```

ESLint

[error] Insert `⏎·····`

(prettier/prettier)